public/rooms/loop/predialogue.html

```diff
@@ -4250,64 +4282,61 @@ <h2 id="lotsnlots">Talk to Loop even more</h2>
+        </p>
 
-                    <br>
+        <br>
 
-                    <p class="dialogue-line">
-                        <span class="dialogue-head">
-                            <span class="dialogue-name">Loop</span>
-                            <span class="dialogue-expression">(teehee2)</span>
-                        </span>
-                        Yes, yes!
-                    </p>
-                </div>
-            </details>
-        </li>
+        <p class="dialogue-line">
+            <span class="dialogue-head">
+                <span class="dialogue-name">Loop</span>
+                <span class="dialogue-expression">(teehee2)</span>
+            </span>
+            Yes, yes!
+        </p>
         
-        <li>
-            <details>
-                <summary>If you have found the Bright Flower before</summary>
-                
-                <div class="dialogue">
-                    <p class="dialogue-line">
-                        (Is there...?)
-                    </p>
+    </div>
+    
+    <br>
+    
+    <p class="dialogue-note">If you have found the Bright Flower before</p>
+    
+    <div class="indented">
+    
+        <p class="dialogue-line">
+            (Is there...?)
+        </p>
 
-                    <br>
-                    
-                    <p class="dialogue-line">
-                        <span class="dialogue-head">
-                            <span class="dialogue-name">Loop</span>
-                            <span class="dialogue-expression">(oh my1)</span>
-                        </span>
-                        How did you not find it already?!?
-                    </p>
+        <br>
 
-                    <p class="dialogue-line">
-                        <span class="dialogue-head">
-                            <span class="dialogue-name">Loop</span>
-                            <span class="dialogue-expression">(hmmm1)</span>
-                        </span>
-                        You should explore Dormont a bit more, stardust! There's a flower you can get, and give it to your party.
-                    </p>
-                    
-                </div>
-            </details>
-        </li>
-    </ul>
+        <p class="dialogue-line">
+            <span class="dialogue-head">
+                <span class="dialogue-name">Loop</span>
+                <span class="dialogue-expression">(oh my1)</span>
+            </span>
+            How did you not find it already?!?
+        </p>
+
+        <p class="dialogue-line">
+            <span class="dialogue-head">
+                <span class="dialogue-name">Loop</span>
+                <span class="dialogue-expression">(hmmm1)</span>
+            </span>
+            You should explore Dormont a bit more, stardust! There's a flower you can get, and give it to your party.
+        </p>
+        
+    </div>
     
-    <br>
+    <!-- [sic] -- they ARE linked the wrong way, i've double and triple checked -->
```

Comment:
disagree
![image](https://github.com/user-attachments/assets/ee5766d0-6f19-43a3-a0a8-c20171659897)
if `RE_foundfloweronce_AL` is OFF - so, if not found flower before, be confused, else, be not confused

Reply:
gonna go fix this myself ^^